assets/src/setup/pages/site-scan/index.js

```diff
@@ -2,11 +2,28 @@
  * WordPress dependencies
```

Comment:
Changes to this file are for testing only. The full site scan will be built out in phase 2.